tests/README.md

```
@@ -39,21 +39,33 @@ npm run playwright:show-report
-2. **Frontend dev server is running**: `npm run dev`
-3. **Environment variables** - Create `.env.local` with:
+2. **Environment variables** - Create `.env.local` with:
    ```env
    REACT_CARE_API_URL=http://127.0.0.1:9000
    ```
 
+Note: The Playwright configuration is set to automatically start a production server using `npm run preview` via the `webServer` command. You do not need to have `npm run dev` running in a separate terminal to execute tests.
+
 ## Test Structure
 
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)